src/memray/reporters/templates/__init__.py

```diff
@@ -13,9 +13,11 @@
 
 @lru_cache(maxsize=1)
 def get_render_environment() -> jinja2.Environment:
-    return jinja2.Environment(
+    env = jinja2.Environment(
         loader=jinja2.PackageLoader("memray.reporters"),
     )
+    env.policies["json.dumps_kwargs"] = {"sort_keys": True, "separators": (",", ":")}
```

Comment:
jesus

Reply:
This change alone saves us about 2 MB on the size of the nbody example program's flamegraph - apparently we had 2 MB of useless spaces! 😅